android/config.json

```diff
@@ -0,0 +1 @@
+{"copyFiles":["IronSourcePlugin.java"],"injectionXML":"manifest.xml","injectionXSL":"manifest.xsl","proguardXML":"proguard.xml","injectionTealeafModuleGradleXML":"gradletealeaf.xml","injectionGradleClasspathXML":"gradleclasspath.xml"}
```

Comment:
Can this be made more readable ?

Reply:
This requires some formatting during prebuild hook in js because this files are copied for every providers and the merged together.

Reply:
okay, understood. we can ignore it